tests/test_fieldtypes.py

```diff
@@ -9,13 +9,9 @@
 
 import flow.record.fieldtypes
 from flow.record import RecordDescriptor, RecordReader, RecordWriter
-from flow.record.fieldtypes import (
-    PATH_POSIX,
-    PATH_WINDOWS,
-    fieldtype_for_value,
-    net,
-    uri,
-)
+from flow.record.fieldtypes import PATH_POSIX, PATH_WINDOWS
+from flow.record.fieldtypes import datetime as dt
+from flow.record.fieldtypes import fieldtype_for_value, net, uri
```

Comment:
Why this change? isort or black don't like the tuple formatting when `as` is used?

Reply:
Yes, that was isort I'm afraid...